packages/@uppy/aws-s3/types/index.d.ts

```diff
@@ -10,8 +9,20 @@ export interface AwsS3UploadParameters {
     headers?: { [type: string]: string }
 }
 
-export interface AwsS3Options extends AwsS3MultipartOptions {
-    /** @deprecated future versions of this plugin will use the Expires value from the backend */
+export interface AwsS3Options {
+    id?: string
+    companionUrl?: string | null
+    companionHeaders?: Record<string, string>
+    allowedMetaFields?: Array<string> | null
+    getUploadParameters?: (file: UppyFile) => MaybePromise<{
+      method: "PUT" | "POST"
+      url: string
+      fields: Record<string, string> | null
+      headers: Record<string, string>
+    }>
+    limit?: number
+    getResponseData?: (responseText: string, response: XMLHttpRequest) => void
+    locale?: { strings: Record<string, string> }
```

Comment:
On looking at some of the locale examples this looks wrong

https://github.com/transloadit/uppy/blob/main/packages/%40uppy/locales/src/fr_FR.js
https://github.com/transloadit/uppy/blob/main/packages/%40uppy/locales/src/fi_FI.js

I will update but if there is better type information on this please let me know.